Assumes 'tar' is GNU tar, and passes inappropriate command-line options.

--- cmake/modules/KDevPlatformMacros.cmake.orig	2016-12-31 01:03:07.457715000 +0100
+++ cmake/modules/KDevPlatformMacros.cmake	2016-12-31 01:04:32.112215000 +0100
@@ -52,8 +52,8 @@
     else(WIN32)
         add_custom_command(OUTPUT ${_template}
             COMMAND tar ARGS -c -C ${CMAKE_CURRENT_SOURCE_DIR}/${_templateName}
-                --exclude .kdev_ignore --exclude .svn --mode=go=rX,u+rw,a-s --owner=root
-                --group=root --numeric-owner -j -f ${_template} .
+                --exclude .kdev_ignore --exclude .svn --uname=root
+                --gname=root --numeric-owner -j -f ${_template} .
             DEPENDS ${_deps}
         )
     endif(WIN32)
